Lib/fontTools/ttLib/tables/otTables.py

```diff
@@ -1187,6 +1188,42 @@ def preWrite(self, font):
 		}
 
 
+class LookupList(BaseTable):
+	@property
+	def table(self):
+		for l in self.Lookup:
+			for st in l.SubTable:
+				if type(st).__name__.endswith("Subst"): return "GSUB"
```

Comment:
sorry to always nag about style, but can you please stick to indenting all `if` blocks over multiple lines, no matther how short, thanks :)

Reply:
Happy to do this, but this would all be *so* much easier if we could run black on the codebase. If we're going to insist on a code style, let's make the repo reflect that, and then running black on any changes afterwards will produce manageable diffs.

After this is in, I'm going to put in a set of PRs reformatting the parts of the code that I plan to work on.

Reply:
I've been hesitant reformatting all codebase with black so far, but now I convinced myself that it's worth doing it once and for all so we don't waste more time on these styling debates. But this only makes sense if at the same time we enforce it for all subsequent commits (e.g. `black --check --diff` in CI before running test suite).
Let's move the "blacken everything" to its own issue/PR.